XVim/XVimTester.m

```diff
@@ -103,6 +103,9 @@ - (NSArray*)categories{
 #pragma clang diagnostic pop
         }
     }
+    // category names need to be sorted alphabetically
+    NSArray* sorted = [arr sortedArrayUsingSelector:@selector(compare:)];
+    return sorted;
     return arr;
```

Comment:
Looks like this line can be removed? Also you could just return the sorted call without assigning the temporary variable.